game/menu.cc

```diff
@@ -31,10 +29,10 @@ bool MenuOption::isActive() const {
 }
 
 
-Menu::Menu(): dimensions(), m_open(true) { clear(); }
+Menu::Menu(): dimensions(), m_open(true), root_options(std::make_shared<MenuOptions>()) { clear(); }
```

Comment:
same, If yo ureally want to stress this is set to nullptr, set it at nullptr at its declaration.